src/XIVLauncher.Common.Unix/Compatibility/WineSettings.cs

```diff
@@ -24,10 +25,11 @@ public class WineSettings
 
     public DirectoryInfo Prefix { get; private set; }
 
-    public WineSettings(WineStartupType? startupType, string customBinPath, string debugVars, FileInfo logFile, DirectoryInfo prefix, bool? esyncOn, bool? fsyncOn)
+    public WineSettings(WineStartupType? startupType, string customBinPath, string dxvkHudCustomString, string debugVars,  FileInfo logFile, DirectoryInfo prefix, bool? esyncOn, bool? fsyncOn)
```

Comment:
A bit nit-picky but there is an extra space before FileInfo here (well and it is better to not have dxvk stuff here as well)